scripts/fonts.json

```diff
@@ -0,0 +1,402 @@
+{
+  "font-families": {
+    "Gothic A1": ["regular", "700"],
+    "M PLUS Rounded 1c": ["regular", "700"],
+    "Noto Sans": ["regular", "700", "italic", "700italic"],
+    "Noto Sans Arabic": ["regular", "700"],
+    "Noto Sans Armenian": ["regular", "700"],
+    "Noto Sans Bengali": ["regular", "700"],
+    "Noto Sans Devanagari": ["regular", "700"],
+    "Noto Sans Ethiopic": ["regular", "600"],
+    "Noto Sans Georgian": ["regular", "700"],
+    "Noto Sans Gurmukhi": ["regular", "700"],
+    "Noto Sans Hebrew": ["regular", "700"],
+    "Noto Sans Kannada": ["regular", "700"],
+    "Noto Sans Lao": ["regular", "700"],
+    "Noto Sans Malayalam": ["regular", "700"],
+    "Noto Sans Myanmar": ["regular", "700"],
+    "Noto Sans Tamil": ["regular", "700"],
+    "Noto Sans Telugu": ["regular", "700"],
+    "Noto Sans Thai": ["regular", "700"]
```

Comment:
Note that the SIL Open Font License allows the font’s designer to specify a “Reserved Font Name” that others can’t use as the font’s name when distributing the font or any subset of it. According to the [Google Fonts attribution page](https://fonts.google.com/attribution), some of the available fonts do specify a reserved font name, but none of them are listed here. I think we should be fine in any case, because all the stylesheet knows is that it’s called “Americana”.